server/src/main/java/org/opensearch/tasks/TaskInfo.java

```diff
@@ -281,6 +328,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             resourceStats.toXContent(builder, params);
             builder.endObject();
         }
+        if (cancelledAt != -1) {
+            builder.field("cancelled_at_millis", cancelledAt);
```

Comment:
Consider adding this as a human readable field. This shall automatically render a human readable field when `human=true` query parameter is set.

```suggestion
            builder.humanReadableField("cancelled_at_millis", "cancelled_at", new TimeValue(cancelledAt));
```

Reply:
Okay. I had considered this earlier but was not sure. Added now.